server/game/GameActions/UnenrageAction.js

```diff
@@ -0,0 +1,22 @@
+const CardGameAction = require('./CardGameAction');
+
+class UnenrageAction extends CardGameAction {
```

Comment:
I'm not sure this needs to be a game action, it should probably just be part of the fight action